### ISSUE TITLE
Entrypoint in poll server

### ISSUE THREAD

**Issue by fnwirth** [medizininformatik-initiative/distributed-analysis_datashield-qp#4]
The entry point in the poll server [dockerfile](https://github.com/medizininformatik-initiative/distributed-analysis_datashield-qp/blob/0eaa05233185fc46deb986780e1bce4486dd355b/deploy/opal_poll/docker-compose.yml)  of the deploy folder is a dummy one.
In comparisson the [dockerfile](https://github.com/medizininformatik-initiative/distributed-analysis_datashield-qp/blob/0eaa05233185fc46deb986780e1bce4486dd355b/ds_poll/Dockerfile) of the folder ds_poll starts an entry point script which in turn start the poll server.
I guess the entry point in the deploy/poll server folder is also supposed to start the poll server?

**Comment by juliangruendner** [comment]
yes - it has to be removed. maybe its best to delete the deploy folder or move it to another branch, as it is outdated
